x-pack/plugins/apm/public/components/fleet_integration/apm_agents/agent_instructions_mappings.ts

```diff
@@ -55,7 +54,6 @@ export const ApmAgentInstructionsMappings: Array<{
     title: 'Java',
     variantId: 'java',
     createAgentInstructions: createJavaAgentInstructions,
-    AgentRuntimeAttachment: JavaRuntimeAttachment,
```

Comment:
We may want to just comment with a `// TODO` to add it back

Reply:
That's totally fine, if you prefer. I deleted it because it's in git.